ui/app/AppLayouts/Profile/views/wallet/ManageTokensView.qml

```diff
@@ -128,10 +133,74 @@ ColumnLayout {
     }
 
     Component {
-        id: supportedTokensListPanel
-        SupportedTokenListsPanel {
-            sourcesOfTokensModel: root.sourcesOfTokensModel
-            tokensListModel: root.tokensListModel
+        id: advancedTab
+        ColumnLayout {
+            spacing: 0
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.topMargin: 18
+                Layout.bottomMargin: 18
+                text: qsTr("Token lists")
+                color: Theme.palette.baseColor1
+            }
+            SupportedTokenListsPanel {
```

Comment:
It looks like the layout of the token list delegate has changed slightly. It's not related to this pr but would be nice if you also add the changes in here since they are really small changes:

1. Subtitle text is now 13px
2. Button `View` doesn't have background by default.

<img width="592" alt="Screenshot 2024-01-11 at 10 12 08" src="https://github.com/status-im/status-desktop/assets/97019400/b3f33312-bb52-4536-84c3-65a36b9cb8cc">

Reply:
OK, I can do that

Reply:
Fixed

![image](https://github.com/status-im/status-desktop/assets/5377645/7083bd2b-ce4f-423b-a46f-2a2122ef7ed0)